### ISSUE TITLE
Gnome 42 and Ubuntu 22.04

### ISSUE THREAD

**Issue by ARTEMON06** [vinceliuice/WhiteSur-gtk-theme#572]
A new version of Gnome is coming out soon.
Is it worth updating your OS to a new one after the release? Do you think your theme will work, are you ready for a new version of Gnome? Thanks!

**Comment by vinceliuice** [comment]
If Gnome not gona support third-party themes, then I think I have to hack libadwaita to support this theme

**Comment by 7system7** [comment]
Btw, some applications does not use the system theme in GNOME 41 too. In this case, the "system theme" is the WhiteSur obvoiusly.

![Screenshot from 2022-03-16 14-10-30](https://user-images.githubusercontent.com/3964568/158597187-f81763df-d280-40c3-83c7-78c31220f631.png)

**Comment by 7system7** [comment]
Is there any issue or discussion to follow, how is the story going? :)

**Comment by ebanDev** [comment]
Hey ! Any news ?

**Comment by vinceliuice** [comment]
@ebanDev  run `./install.sh -l` to make libadwaita use gtk-4.0 theme

**Comment by ebanDev** [comment]
Oh wow it works ! Thanks !! :D

**Comment by ebanDev** [comment]
Is it normal libadwaita apps are in dark theme 🤔

**Comment by vinceliuice** [comment]
You can change to any theme, just run ./install.sh again, for example, `./install.sh -c light -l` to install the light one for libadwaita

**Comment by ebanDev** [comment]
Yeaaah but, it doesn't seem to change according to system theme :/

**Comment by vinceliuice** [comment]
Yes libadwaita not support changing theme according to theme setting, this is just a way to set all gtk 4.0 use a theme

**Comment by ebanDev** [comment]
Uh okay :/ That's a pity, libadwaita sucks :(

**Comment by Niron68** [comment]
> @ebanDev run `./install.sh -l` to make libadwaita use gtk-4.0 theme

i get `ERROR: Unrecognized installation option '-l'.` with the -l options

**Comment by ebanDev** [comment]
> > @ebanDev run `./install.sh -l` to make libadwaita use gtk-4.0 theme
> 
> i get `ERROR: Unrecognized installation option '-l'.` with the -l options

git pull ?

**Comment by Niron68** [comment]
> git pull ?

Ok, i was trying with the last release. Ty, i cloned the project and it works perfectly :)

**Comment by LennonMcCartney** [comment]
I have tried this but for me it says it works in the terminal but then doesn't actually do anything and it still uses the default libadwaita theme. Why do the people making gnome have to be this annoying about themes!

I have tried restarting haha. This is with fedora 36 if that matters
